src/main/java/com/jsyn/apps/AboutJSyn.java

```diff
@@ -4,9 +4,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Adding "main/java" to the path will break lots of web links that point to source code.
Why is this necessary?

Reply:
The file structure is standard for all Gradle and Maven projects ([from the Gradle site](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html)). You can reconfigure and change it if you really want, however it may cause some unintended side effects (primarily relating to maven publishing) and of course breaking of convention. If the links reference the repository revision they won't change, otherwise, they will be broken.